Makefile

```diff
@@ -26,6 +26,22 @@ securedrop-workstation-grsec: ## Builds Debian metapackage for Qubes Workstation
 install-deps: ## Install initial Debian packaging dependencies
 	./scripts/install-deps
 
+.PHONY: requirements
+requirements: ## Creates requirements files for the Python projects
+	./scripts/create-requirements
+	./scripts/update-requirements
+
+.PHONY: build-wheels
+build-wheels: syncwheels ## Builds the wheels and syncs to the localwheels directory
+	./scripts/build-sync-wheels -p ${PKG_DIR}
+	./scripts/sync-sha256sums
+	./scripts/createdownloadurls.py > wheelsurls.txt
```

Comment:
Finally at this stage we can push the modified wheels and updated PyPI index files to S3, so we can again add a note (just print to the user "run aws ...")